src/cli.rs

```diff
@@ -52,23 +51,86 @@ pub fn group_subcommands(stdout: &str) -> (Vec<&str>, Vec<&str>) {
     (cross, host)
 }
 
-pub fn fmt_subcommands(stdout: &str, msg_info: MessageInfo) -> Result<()> {
+pub fn fmt_subcommands(stdout: &str, msg_info: &mut MessageInfo) -> Result<()> {
     let (cross, host) = group_subcommands(stdout);
     if !cross.is_empty() {
-        shell::print("Cross Commands:", msg_info)?;
+        msg_info.print("Cross Commands:")?;
         for line in cross.iter() {
-            shell::print(line, msg_info)?;
+            msg_info.print(line)?;
         }
     }
     if !host.is_empty() {
-        shell::print("Host Commands:", msg_info)?;
+        msg_info.print("Host Commands:")?;
         for line in cross.iter() {
-            shell::print(line, msg_info)?;
+            msg_info.print(line)?;
         }
     }
     Ok(())
 }
 
+fn is_verbose(arg: &str) -> bool {
```

Comment:
This corrects the behavior so `-vvvvv` and similar are accepted as arguments to get the verbosity. Previously this was passed to `cargo`, but not correctly processed for `cross` itself.

Reply:
Nice, was meaning to fix this myself